### PR TITLE
Bug 1836153 - Changed line 114 in compare_tasks.py 

### PR DESCRIPTION
**Pull request by tfdeleon**
Bug-1836153 - Changed python version from 3.9 -> 3. Changed line 114 in compare_tasks.py from 'th_instancely' to "instance"

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ repos:
     rev: 22.10.0
     hooks:
       - id: black
-        language_version: python3.9
+        language_version: python3
```
Comment (jmaher) [inline .pre-commit-config.yaml:24]: we shouldn't change this, the more specific we can get the better- if there is a reason to change it, let me know

**Review by jmaher — CHANGES_REQUESTED** [review]
please address the comment about python 3.9 vs python 3

**Comment by tfdeleon** [comment]
﻿﻿Hey Joel, I hope your trip went well ! Sorry about resubmitting without changing anything. We've been in our project week for our bootcamp and I was just trying to see what you were referring to in the bug and I accidentally clicked "Edit bug" thinking it'll show me what I did wrong. But I accidentally submitted. I just finished submitting my project and it went well! Learned so much from this experience. I will get those bugs fixed this weekend. -TimOn Jun 7, 2023, at 8:51 AM, Joel Maher ***@***.***> wrote:﻿
@jmaher requested changes on this pull request.

please address the comment about python 3.9 vs python 3

**Comment by jmaher** [comment]
all is fixed :)

**Pull request merged**